.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug or performance issue report
```

Comment:
Please capitalize the header row of the table: about, title, etc.

Please add a period to the end of full sentences: Create a report...

Reply:
Done.